app/views/posts/show.html.erb

```diff
@@ -13,6 +13,9 @@
     <ul class="container">
       <% @post.comments.each do |comment| %>
         <li><%= comment.author.name %>: <%= comment.text %></li>
+        <% if can? :destroy, Comment %>
+          <%=button_to "delete", user_post_comment_path(current_user, @post, comment), method: :delete %>
+        <% end %>
```

Comment:
- [ ] I created two accounts in two browsers, and I was able to delete another person's comment, kindly fix this issue. I think it is not even good to show a delete button to users who are not eligible to delete a comment.

![roles](https://user-images.githubusercontent.com/53124548/224068497-5cf7ba47-2d67-4183-8d5b-31deda76dc6d.png)

![post-delete](https://user-images.githubusercontent.com/53124548/224068585-ba97bb0d-2d1e-4e9c-9e24-0dc9faff685c.png)